conseil-common-testkit/src/main/scala/tech/cryptonomic/conseil/common/testkit/InMemoryDatabaseSetup.scala

```diff
@@ -14,16 +16,42 @@ trait InMemoryDatabaseSetup {
     }.toSeq
 
   // Keep in mind that this is sorted to preserve key consistency
-  def fixtures: Seq[Fixture[_]] = register.flatMap(_._2).toSeq
+  def fixtures: Seq[Fixture] = register.flatMap(_._2).toSeq
 
-  def registerSchema(schema: String, fixtures: Seq[Fixture[_]]): Unit = register += schema -> fixtures
+  def registerSchema(schema: String, fixtures: Seq[Fixture]): Unit = register += schema -> fixtures
 
-  /*** Table fixture is needed to run away from type madness in Slick */
-  case class Fixture[U](query: TableQuery[_ <: Table[U]]) {
+  /*** Fixture is needed to run away from type madness in Slick */
+  trait Fixture {
+    def create: DBIOAction[Unit, NoStream, Effect.Schema]
+    def delete: DBIOAction[Int, NoStream, Effect.Write]
+  }
+
+  /** Companion object for Fixture trait */
+  object Fixture {
+
+    /** Creates fixture for tables */
+    def table[U](query: TableQuery[_ <: Table[U]]): Fixture = TFixture(query)
+
+    /** Creates fixture for views */
+    def view(createSql: String): Fixture = view(Iterable(createSql))
+    def view(createAction: SqlAction[Int, NoStream, Effect]): Fixture = view(createAction.statements)
+    def view(createStatements: Iterable[String]): Fixture = VFixture(createStatements)
+  }
+
+  /*** Represents fixture for common slick tables */
+  case class TFixture[U](query: TableQuery[_ <: Table[U]]) extends Fixture {
     def create: DBIOAction[Unit, NoStream, Effect.Schema] = query.schema.create
     def delete: DBIOAction[Int, NoStream, Effect.Write] = query.delete
   }
 
+  /*** Represents fixture for custom slick views */
+  case class VFixture(createSql: Iterable[String]) extends Fixture {
```

Comment:
actually this is a fixture for any SQL statement, not only views, maybe we should change the name to more generic?

Reply:
as long as the commodity `Fixture.table/view` methods are there, I agree that this can be made general. though I wonder if it makes sense to run `.create` on something that, I don't know, reset some value or configures a db feature...

Reply:
Since internally it uses a ddl, what do you have in mind? Sequences, functions?

Reply:
I would see it more like: Create tables and run some queries (that can contain view, function or whatever) but it's up to Adrian

```scala
Fixture.table(Tables.Blocks)
Fixture.sql(Views.AccountsViewSql)
```

Reply:
Since there is no special need for creating functions or anything like that, I'm going to stick to what we have now. As you can see, it would be (probably) quite easy to do, but since creating anything different than view is not tested and we do not have time for that, I'm not going to extend that only because it is possible to do.

I'm not saying it is not possible or bad, only that it is not needed yet :)